projects/core/components/link/link.style.less

```diff
@@ -112,7 +112,7 @@
 }
 
 .t-content {
-    display: inline-block;
+    display: inline;
```

Comment:
This is unnecessary, it's already inline - it's a `span`.

Reply:
fixed